### PR TITLE
Add status field to all istio crds

### PR DESCRIPTION
**Pull request by therealmitchconnors**
WARNING: tests are incapable of failing in this state.  Please review carefully.

### CODE REVIEW

File: cmd/kubetype-gen/generators/types.go [hunk 1]
```diff
@@ -91,6 +92,7 @@ type $.KubeType.Type|public$ struct {
 	// +optional
 	Spec $.RawType|raw$ ` + "`" + `json:"spec,omitempty" protobuf:"bytes,2,opt,name=spec"` + "`" + `
 
+	Status $.IstioStatus|raw$ ` + "`" + `json:",inline"` + "`" + `
```
Comment (jasonwzm) [inline cmd/kubetype-gen/generators/types.go:95]: Why do we want inline for marshalling?

File: cmd/kubetype-gen/testdata/test_output/defaults/group/version/doc.go [hunk 2]
```diff
@@ -17,4 +17,5 @@
 // Package has auto-generated kube type wrappers for raw types.
 // +k8s:openapi-gen=true
 // +k8s:deepcopy-gen=package
+// +groupName=group
```
Comment (jasonwzm) [inline cmd/kubetype-gen/testdata/test_output/defaults/group/version/doc.go:20]: Do you know why groupName is generated here?

Comment (therealmitchconnors) [reply cmd/kubetype-gen/testdata/test_output/defaults/group/version/doc.go:20]: No, I don't see where that came from...

**Comment by therealmitchconnors** [comment]
The blocking PR in API has been merged.  This is ready to go pending review.  @jasonwzm PTAL.

**Pull request merged**